feat(sdk-verification)!: Implementing DIF Presentation exchange protocol 2.0 for SDK to SDK OOB verification (#196)
input-output-hk/atala-prism-wallet-sdk-ts: docs/examples/index.md

# Identus Edge SDK TS Examples

> Here you can find basic implementations and tutorials for the Identus Edge Agent JS SDK, but most things run cross platform with minor differences.

Reviewer: ```suggestion
Here, we have the basic implementations and tutorials for the Identus Edge Agent JS SDK, but most things run cross-platform with minor differences.
```

Author: ok

## Tutorials

### Cross platform DK Verification

> Start a verification flow from an Edge Agent (holder wallet) and another holder to request (a did) to send you a credenital proof for a specific issuer and specific claims.

Author: Changed in next commit, solving
